cmd/example-mysql/main.go

```diff
@@ -171,7 +171,7 @@ func configureTilesReadAPI(mux *http.ServeMux, storage *mysql.Storage) {
 			return
 		}
 
-		entryBundle, err := storage.ReadEntryBundle(r.Context(), index/256)
+		entryBundle, err := storage.ReadEntryBundle(r.Context(), index)
```

Comment:
Can this be split into its own PR? Each PR should do a single purpose unless there's a clear overlap. I don't see any overlap here?